Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/README.md

```diff
@@ -88,215 +53,205 @@ RBAC Permissions Required - org.alerts.dismiss: EXECUTE
 | CarbonBlackEEDR.Alert.State | String | The alert state. | 
 
 
-##### Command Example
-```!cb-eedr-alert-workflow-update alert_id=A28C720DCBCD66333A624893AB1E0FE9 state=open```
+#### Command Example
+``` ```
```

Comment:
you have deleted the example of this and the following commands, please re add them